src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final int INDEX_OPERATION_TYPE = 0;
+    public static final int INDEX_AMOUNT = 1;
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+    public static final String SEPARATOR = ",";
+    private int supplyAmount = 0;
+    private int buyAmount = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        readData(fromFileName);
+        writeData(toFileName);
+    }
+
+    private void readData(String fromFileName) {
```

Comment:
readData should return something that it read. 
```suggestion
    private ? readData(String fromFileName) {
```